### PR TITLE
OCPQE-18877: add minimal permission config for azure-ipi-disk-encryption-set

### PR DESCRIPTION
**Pull request by jinyunma**
change multi-nightly job `azure-ipi-disk-encryption-set-xxx` to `azure-ipi-des-minimal-permission-xxx` on 4.15+ to cover case [OCP-68607 - [ipi-on-azure] install and destroy cluster configured DES with minimum permisison](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-68607)

### CODE REVIEW

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-azure-ipi-des-minimal-permission-arm-f14

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-azure-ipi-des-minimal-permission-arm-f14

**Comment by jinyunma** [comment]
cc @jianlinliu @gpei to review

File: ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/disk-encryption-set/minimal-permission/cucushift-installer-rehearse-azure-ipi-disk-encryption-set-minimal-permission-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+workflow:
+  as: cucushift-installer-rehearse-azure-ipi-disk-encryption-set-minimal-permission
+  steps: 
+    pre:
+    - ref: azure-provision-service-principal-minimal-permission
+    - chain: cucushift-installer-rehearse-azure-ipi-disk-encryption-set-provision
+    post:
+    - chain: cucushift-installer-rehearse-azure-ipi-disk-encryption-set-deprovision
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/disk-encryption-set/minimal-permission/cucushift-installer-rehearse-azure-ipi-disk-encryption-set-minimal-permission-workflow.yaml:8]: we need `azure-deprovision-sp-and-custom-role` step here?

Comment (jinyunma) [reply ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/disk-encryption-set/minimal-permission/cucushift-installer-rehearse-azure-ipi-disk-encryption-set-minimal-permission-workflow.yaml:8]: good catch, fixed.

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.15-installer-rehearse-azure

**Comment by jianlinliu** [comment]
cc @liangxia to review.

**Pull request merged**